.sbtopts

```diff
@@ -1,7 +1,7 @@
--J-Xmx2G
--J-Xms1G
--J-Xss2M
+-J-Xmx4g
+-J-Xms1g
+-J-Xss6M
 -Dfile.encoding=UTF8
 -Djava.awt.headless=true
 -Dsun.io.serialization.extendedDebugInfo=true
--Djava.library.path=/usr/local/lib
+-Djava.library.path=/usr/local/lib:/usr/local/opt/gdal2/lib
```

Comment:
On my macOS i have GDAL installed on `/usr/local/lib` O: sounds more like a user-side setting

Reply:
Removed it.